lib/ht_sip_validator/validation/meta_yml.rb

```diff
@@ -38,18 +38,38 @@ def validate
       end
 
       class RequiredKeys < Validation::Base
-        REQUIRED_KEYS = %w(capture_date)
+        REQUIRED_KEYS = %w(capture_date).freeze
         def validate
           REQUIRED_KEYS.each do |key|
-            unless @sip.meta_yml.has_key?(key)
-              record_error(
-                validation: :has_field,
-                human_message: "Missing required key #{key} in meta.yml",
-                extras: { filename: "meta.yml",
-                          field: key }
-              )
-
-            end
+            next if @sip.meta_yml.key?(key)
+            record_error(
+              validation: :field_presence,
+              human_message: "Missing required key #{key} in meta.yml",
+              extras: { filename: "meta.yml",
+                        field: key }
+            )
+          end
+
+          super
+        end
+      end
+
+      class UnknownKeys < Validation::Base
+        require "set"
+        KNOWN_KEYS = %w(capture_date scanner_make scanner_model scanner_user
```

Comment:
The `MetaYml` file doesn't have its own representation (or didn't, I haven't looked at #42 yet), so I'm not sure where you're suggesting these would go?